pkg/playground/react-patterns.js

```diff
@@ -75,7 +75,7 @@ const onStandardDemoClicked = (staticError) => {
     const dialogProps = {
         title: "This shouldn't be seen",
         body: React.createElement(PatternDialogBody, { clickNested: onStandardDemoClicked }),
-        static_error: staticError,
+        error: staticError,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19505-20231019-074956-e634b41b-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)